lxd/device/device_utils_usb.go

```diff
@@ -0,0 +1,66 @@
+package device
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+)
+
+// devPath is the path where USB devices can be enumerated.
+const devPath = "/sys/bus/usb/devices"
+
+// USBDevice represents the properties of a USB device.
+type USBDevice struct {
```

Comment:
I wonder if we should consider pushing that data into the resources API, tracking USB devices in there directly as those fields look very similar to what we do for GPU, disks, ...

Probably not worth doing right now but if we see some demand for users trying to figure out USB devices on their systems, then moving to resources API would be the way to go.